### PR TITLE
Fix preschool mentors assigning paragraph 

### LINKED ISSUE #184: Pre school description typo
## 🐛 Describe the Bug
typo in the pre school course description

## 📸 Screenshots

Image

### PR DESCRIPTION
**Pull request by Tedzury**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description

Removed displaying the paragraph about assigning mentors at preschool page. I suppose that way of fixing that issue as a bad practice, but rendering logic of that component is overcomplicated, so I chose the most univasive way of fixing.

## Related Tickets & Documents

- Related Issue #184 
- Closes #184 

## Screenshots, Recordings

Screenshot 2024-05-09 163625

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

File: src/features/training-program/training-program.tsx [hunk 1]
```diff
@@ -17,20 +17,23 @@ export const TrainingProgram = ({ courseName, type }: TrainingProgramProps) => {
...
+      ? content.filter((component) => component.type.name !== 'Subtitle')
+      : content;
   return (
     <section className="training-program container">
       <div className="training-program content column-2">
         <div className="left">
           <Title text={title} hasAsterisk />
 
-          {content.map((component, index) => cloneElement(component, { key: index }))}
+          {filteredContent.map((component, index) => cloneElement(component, { key: index }))}
```
Comment (andron13) [inline src/features/training-program/training-program.tsx:36]: To me index is not a property that is good in key. Do we have a unique Title?

File: src/features/training-program/training-program.tsx [hunk 2]
```diff
@@ -17,20 +17,23 @@ export const TrainingProgram = ({ courseName, type }: TrainingProgramProps) => {
...
 
           <Button label={buttonLabel} href={course?.enroll} />
```
Comment (andron13) [inline src/features/training-program/training-program.tsx:38]: Button with href?

File: src/features/training-program/training-program.tsx (hunk 1 above)
Comment (Tedzury) [reply src/features/training-program/training-program.tsx:36]: Content is just an array of React components, they don't even have a title, or anything else. That is what I mentioned in the description at PR - very strange and complicated logic, at least for me. 🤷‍♂️ But, if we look at it from the other side - these component never changing, once rendered, so, maybe, index as key is not a problem at all.

File: src/features/training-program/training-program.tsx (hunk 2 above)
Comment (Tedzury) [reply src/features/training-program/training-program.tsx:38]: As I see - button component contains anchor tag and leads to registration for course.

File: src/features/training-program/training-program.tsx [hunk 3]
```diff
@@ -17,20 +17,23 @@ export const TrainingProgram = ({ courseName, type }: TrainingProgramProps) => {
...
-
   const course = data as Course;
 
   const { title, content, image } = contentMap[courseName];
 
   const buttonLabel = courseName === 'react ru' ? 'Записаться' : 'Register';
 
+  const filteredContent =
+    type === 'Pre-school'
+      ? content.filter((component) => component.type.name !== 'Subtitle')
```
Comment (SpaNb4) [inline src/features/training-program/training-program.tsx:28]: Could you leave a comment inside `Subtitle.tsx` letting us know that if the component name is going to be changed, it should be updated here as well?

File: src/features/training-program/training-program.tsx
Comment (SpaNb4) [inline src/features/training-program/training-program.tsx:1]: Pretty tricky way to do it, but looks like the least time-consuming

**Review by Quiddlee — APPROVED** [review]

**Comment by andron13** [comment]
This branch has conflicts that must be resolved
Use the [web editor](https://github.com/rolling-scopes/site/pull/190/conflicts) or the  to resolve conflicts.
Conflicting files
src/app/components/subtitle/subtitle.tsx

**Review by andron13 — APPROVED** [review]

**Pull request closed without merging**